### PR TITLE
Schedule glean and legacy telemetry recipes

### PR DESCRIPTION
**Pull request by ANich**
Nightly CI job to run glean and legacy telemetry ingestion. Also update to use latest python orb.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,8 +15,51 @@ jobs:
...
       - unit-tests
+  nightly:  # Sync custom integration sources
+    jobs:
+      - unit-tests
+      - glean-source:
+          requires:
+            - unit-tests
+      - legacytelemetry-source:
+          requires:
+            - unit-tests
```
Comment (mikaeld) [inline .circleci/config.yml:58]: Modify the jobs instances to include the new parameter

```suggestion
  nightly:  # Sync custom integration sources
    jobs:
      - unit-tests
      - datahub-ingest:
          name: glean-source
          recipe: recipes/glean_recipe.dhub.yaml
          requires:
            - unit-tests
      - datahub-ingest:
          name: legacytelemetry-source
          recipe: recipes/legacy_recipe.dhub.yaml
          requires:
            - unit-tests
```

**Review by mikaeld — CHANGES_REQUESTED** [review]

**Comment by mikaeld** [comment]
Also forgot to mention this, but you should use the latest `python: circleci/python@2.1.1`

File: .circleci/config.yml (hunk 1 above)
Comment (ANich) [reply .circleci/config.yml:58]: Makes sense to me. Curious what does the `name` parameter do in the workflow? I don't see it used in the job def'n

**Review by mikaeld — APPROVED** [review]
:rocket:

**Pull request merged**